src/api/menu/MenuAPISpec.js

```diff
@@ -20,7 +20,7 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-import Vue from 'vue';
+import { nextTick } from 'vue';
```

Comment:
`Vue` was only available to us because of `vue/compat`. So now, only import the function itself